### PR TITLE
[feature/NRLF-626-live_rehearsal_feedback] Implement live rehearsal feedback

### PR DESCRIPTION
**Pull request by jaklinger**

### CODE REVIEW

File: nrlf_converter/nrl/tests/data/NRLF-626-relatesTo_with_identifier.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
+{
```
Comment (samgardner1-nhs) [inline nrlf_converter/nrl/tests/data/NRLF-626-relatesTo_with_identifier.json:1]: Have you got  test cases for relatesTo when both identifier and reference are present and when neither are present. Also we got this scenario when running the hard delete pointers script
             "relatesTo" : {
               "code" : null,
               "target" : {
                  "identifier" : {
                     "system" : null,
                     "value" : null
                  }
               }
            },
However, I think this was for superseded status. We could really do with finding out all the relatesTo scenarios for current pointers

**Review by samgardner1-nhs — COMMENTED** [review]
Do we need to fix this error:
error=" Field 'DocumentPointer.class_' was expected but not provided."
I've added a comment to the Jira

File: nrlf_converter/utils/validation/validators.py [hunk 2]
```diff
@@ -122,5 +127,7 @@ def _validate_datetime(obj, date_format: str = None):
         else:
             dt.strptime(_obj, date_format)
     except (ValueError, TypeError):
-        raise InvalidValue(f"Could not parse datetime from '{obj}'.") from None
+        result = R4_DATETIME_REGEX.match(_obj)
```
Comment (samgardner1-nhs) [inline nrlf_converter/utils/validation/validators.py:130]: Not sure about the datetime one. Is there a standard for the datetime format? There were only 2 of these errors in the LR and both were on test patients. We can probably delete both these pointers.

File: nrlf_converter/nrl/tests/data/NRLF-626-relatesTo_with_identifier.json (hunk 1 above)
Comment (jaklinger) [reply nrlf_converter/nrl/tests/data/NRLF-626-relatesTo_with_identifier.json:1]: there's one for when neither are present, but I'll add a test for when both are present. thanks for the feedback

**Review by samgardner1-nhs — DISMISSED** [review]
I've checked 5 relatesTo converter errors from the LR and they were all of the type reference is null and identifier is specified.

**Review by samgardner1-nhs — APPROVED** [review]

**Pull request merged**